app/code/core/Mage/Adminhtml/Model/System/Config/Backend/Baseurl.php

```diff
@@ -26,9 +26,18 @@
  */
 class Mage_Adminhtml_Model_System_Config_Backend_Baseurl extends Mage_Core_Model_Config_Data
 {
+    /**
+     * @return $this
+     * @throws Mage_Core_Exception
+     */
     protected function _beforeSave()
     {
-        $value = $this->getValue();
+        $value = str_replace(' ', '', $this->getValue());
+
+        if ($value === '') {
+            $label = $this->getFieldConfig()->descend('label');
+            Mage::throwException(Mage::helper('core')->__('%s cannot be emtpy.', $label));
```

Comment:
```suggestion
            Mage::throwException(Mage::helper('core')->__('""%s"" is a required value.', $label));
```

this way there's no need to add another translation to the CSV file

Reply:
Thanks.